sirepo/job.py

```diff
@@ -32,7 +32,8 @@
 #: path supervisor registers to receive requests from server
 SERVER_URI = '/server'
 
-AGENT_URI = '/agent'
```

Comment:
I think  this line shouldn't be deleted

Reply:
This was a duplicate definition. See line 30.